llumnix/backends/bladellm/llm_engine.py

```diff
@@ -304,6 +305,7 @@ def instance_info(self) -> InstanceInfo:
 
     async def step(self):
         self.step_counter += 1
+        self.handle_request_barriers()
```

Comment:
why inject_request_barriers is not called in step, while handle_request_barriers is called in step?